backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerClient.java

```diff
@@ -125,6 +125,8 @@ public int processEvents(String playUuid,
         while(true){
             String event = getNextEvent(playUuid, lastEventId);
             if (event == null) {
+                // Reset of the lastEvent because of the callback singleton usage
+                lastEvent = "";
```

Comment:
I'd rather put it before the loop, it's more readable and explicit in case there's ever another break added